build-scripts/gulp/translations.js

```diff
@@ -266,7 +266,6 @@ gulp.task(taskName, function () {
         TRANSLATION_FRAGMENTS.forEach((fragment) => {
           delete data.ui.panel[fragment];
         });
-        delete data.supervisor;
```

Comment:
So we are now shipping all supervisor localizations to the frontend, always, just for the few ingress error messages? Doesn't it make more sense to make a separate section for those and duplicate them with lokalise keys?